keras2onnx/common/data_types.py

```diff
@@ -3,161 +3,5 @@
-        return onnx_proto.TensorProto.INT32
-
-
-class Int64TensorType(TensorType):
-    def __init__(self, shape=None, doc_string=''):
-        super(Int64TensorType, self).__init__(shape, doc_string)
-
-    def _get_element_onnx_type(self):
-        return onnx_proto.TensorProto.INT64
-
-
-class BooleanTensorType(TensorType):
-    def __init__(self, shape=None, doc_string=''):
-        super(BooleanTensorType, self).__init__(shape, doc_string)
-
-    def _get_element_onnx_type(self):
-        return onnx_proto.TensorProto.BOOL
-
-
-class FloatTensorType(TensorType):
-    def __init__(self, shape=None, color_space=None, doc_string=''):
-        super(FloatTensorType, self).__init__(shape, doc_string)
-        self.color_space = color_space
-
-    def _get_element_onnx_type(self):
-        return onnx_proto.TensorProto.FLOAT
-
-
-class DoubleTensorType(TensorType):
-    def __init__(self, shape=None, color_space=None, doc_string=''):
-        super(DoubleTensorType, self).__init__(shape, doc_string)
-        self.color_space = color_space
-
-    def _get_element_onnx_type(self):
-        return onnx_proto.TensorProto.DOUBLE
-
-
-class StringTensorType(TensorType):
-    def __init__(self, shape=None, doc_string=''):
-        super(StringTensorType, self).__init__(shape, doc_string)
-
-    def _get_element_onnx_type(self):
-        return onnx_proto.TensorProto.STRING
-
-
-class DictionaryType(DataType):
-    def __init__(self, key_type, value_type, shape=None, doc_string=''):
-        super(DictionaryType, self).__init__(shape, doc_string)
-        self.key_type = key_type
-        self.value_type = value_type
-
-    def to_onnx_type(self):
-        onnx_type = onnx_proto.TypeProto()
-        if type(self.key_type) in [Int64Type, Int64TensorType]:
-            onnx_type.map_type.key_type = onnx_proto.TensorProto.INT64
-        elif type(self.key_type) in [StringType, StringTensorType]:
-            onnx_type.map_type.key_type = onnx_proto.TensorProto.STRING
-        onnx_type.map_type.value_type.CopyFrom(self.value_type.to_onnx_type())
-        return onnx_type
-
-
-class SequenceType(DataType):
-    def __init__(self, element_type, shape=None, doc_string=''):
-        super(SequenceType, self).__init__(shape, doc_string)
-        self.element_type = element_type
-        self.doc_string = doc_string
-
-    def to_onnx_type(self):
-        onnx_type = onnx_proto.TypeProto()
-        onnx_type.sequence_type.elem_type.CopyFrom(self.element_type.to_onnx_type())
-        return onnx_type
-
-
-def find_type_conversion(source_type, target_type):
-    '''
-    Find the operator name for converting source_type into target_type
-    '''
-    if type(source_type) == type(target_type):
-        return 'identity'
-    elif type(target_type) == FloatTensorType:
-        return 'imageToFloatTensor'
-    else:
-        raise ValueError('Unsupported type conversion from %s to %s' % (source_type, target_type))
+from onnxconverter_common.data_types import *
```

Comment:
Why not fully delete this file? Is it because we may put data types here that are only used by keras2onnx?

Reply:
Yes, and to maintain backwards compatibility.